chain/x/zoracle/internal/keeper/querier.go

```diff
@@ -230,3 +234,73 @@ func queryPending(ctx sdk.Context, path []string, req abci.RequestQuery, keeper
 func queryRequestNumber(ctx sdk.Context, req abci.RequestQuery, keeper Keeper) ([]byte, sdk.Error) {
 	return codec.MustMarshalJSONIndent(keeper.cdc, keeper.GetRequestCount(ctx)), nil
 }
+
+func queryDataSourceByID(ctx sdk.Context, path []string, req abci.RequestQuery, keeper Keeper) ([]byte, sdk.Error) {
+	if len(path) == 0 {
+		return nil, sdk.ErrInternal("must specify the data source id")
+	}
+	id, err := strconv.ParseInt(path[0], 10, 64)
+	if err != nil {
+		return nil, sdk.ErrInternal(fmt.Sprintf("wrong format for data source id %s", err.Error()))
+	}
+
+	dataSource, sdkErr := keeper.GetDataSource(ctx, id)
+	if sdkErr != nil {
+		return nil, sdkErr
+	}
+
+	res, err := codec.MarshalJSONIndent(keeper.cdc, types.NewDataSourceQuerierInfo(
```

Comment:
For all places that use `MarshalJSONIndent` and panic, why don't we use `MustMarshalJSONIndent`?